cli/reference/cmdline/run.md

```diff
@@ -298,6 +298,40 @@ terramate run [options] -- <cmd ...>
 
   Use Terragrunt to generate the Terraform/OpenTofu Plan file.
 
+### TMC: Deployment Targets
+
+**Note:** This is an upcoming experimental feature that is subject to change in the future. To use it now, you must enable the project config option `terramate.config.experiments = ["targets"]`
```

Comment:
```suggestion
**Note:** This is an experimental feature and may change. Enable it by setting the project config option `terramate.config.experiments = ["targets"]`.
```

Reply:
FYI I copy-pasted this from the scripts section, where it uses similar wording. But I use your suggestion anyway.